### ISSUE TITLE
Error: writeStatic has been removed.

### ISSUE THREAD

**Issue by S13F4NX** [yarbsemaj/sveltekit-adapter-lambda#16]
Hi @yarbsemaj,

it seems that since vite 3 the build process is throwing the following error:

```
> Using adapter-serverless
error during build:
Error: writeStatic has been removed. Please ensure you are using the latest version of adapter-serverless
```

Thx
Stefan

**Comment by yarbsemaj** [comment]
Hi @S13F4NX, 
Thank you for your interest in my project!

I've published a new build (0.10.0) to add support for the latest version of SvelteKit (1.0.0-next.380). I've tested it with my demo project, but could you let me know if this resolves your issue. 

Thanks
James

**Comment by S13F4NX** [comment]
Hi @yarbsemaj ,

Many thanks for fixing this issue und also for setting up this project. 👍 Works very well.

Stefan
